bodo/pandas/series.py

```diff
@@ -581,12 +595,46 @@ def __getattribute__(self, name: str, /) -> pt.Any:
             warnings.warn(BodoLibFallbackWarning(msg))
             return object.__getattribute__(pd.Series(self._series).str, name)
 
+    @check_args_fallback(unsupported="none")
+    def join(self, sep):
+        """
+        Join lists contained as elements in the Series/Index with passed delimiter.
+        If the elements of a Series are lists themselves, join the content of these lists using
+        the delimiter passed to the function.
+        """
+
+        def join_list(l):
```

Comment:
Add GitHub permanent link to Pandas code you looked at for this? I assume it's this one? https://github.com/pandas-dev/pandas/blob/0691c5cf90477d3503834d983f69350f250a6ff7/pandas/core/strings/object_array.py#L292